app/src/main/java/org/mozilla/fenix/home/sessioncontrol/viewholders/CustomizeHomeButtonViewHolder.kt

```diff
@@ -65,11 +63,9 @@ class CustomizeHomeButtonViewHolder(
              * This button will be stylized as a [TertiaryButton] when no wallpaper is selected.
              * Otherwise, the background will use [Wallpaper.cardColorLight] or [Wallpaper.cardColorDark].
              * */
-            Button(
```

Comment:
Let's make `Button` a private function now since we should only expect Primary, Secondary, Tertiary and Destructive Button to be the only usage based on our design system. https://searchfox.org/mozilla-mobile/source/fenix/app/src/main/java/org/mozilla/fenix/compose/button/Button.kt#41